cmake/modules/SwiftSupport.cmake

```diff
@@ -0,0 +1,133 @@
+
+include(CMakeParseArguments)
+
+function(add_swift_target target)
```

Comment:
This seems like a handy, generic function. Are you anticipating moving it somewhere where all the swift-corelibs-* projects could use it, maybe someday?

Reply:
[swift-tools-support-core](https://github.com/apple/swift-tools-support-core) repository is not ready yet but it can be used to hold the shared infrastructure once its open for business!

Reply:
@modocache - you are right, this is a handy, generic function.  I use the same basic functionality in swift-corelibs-libdispatch and am using it in swift-corelibs-foundation too.  Once we have the shared repository, moving this would be great.